cloud-aws-native/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/resource/instance/AwsNativeInstanceResourceBuilder.java

```diff
@@ -127,10 +127,10 @@ public List<CloudResource> build(AwsContext context, CloudInstance cloudInstance
             LOGGER.info("Create new instance with name: {}", cloudResource.getName());
             TagSpecification tagSpecification = awsTaggingService.prepareEc2TagSpecification(awsCloudStackView.getTags(),
                     com.amazonaws.services.ec2.model.ResourceType.Instance);
-            tagSpecification.withTags(
-                    new Tag().withKey("Name").withValue(awsStackNameCommonUtil.getInstanceName(ac, group.getName(), privateId)),
-                    new Tag().withKey("instanceGroup").withValue(group.getName())
-            );
+            tagSpecification.withTags(new Tag().withKey("instanceGroup").withValue(group.getName()));
```

Comment:
This should be allowed to override as well, or document it as reserved by Cloudera.

Reply:
I would vote for the second option as it is not a specific tag that AWS is using, compared to the Name tag.
I can open a DOC ticket for this.

Reply:
Even if we document this, we should fail gracefully, not with a low level error message, so it might be that an extra validation would be needed OR letting the override happen if the value is not used by us strictly.